libraries/protocol/include/xgt/protocol/block_header.hpp

```diff
@@ -28,6 +28,8 @@ namespace xgt { namespace protocol {
       checksum_type                 transaction_merkle_root;
       block_header_extensions_type  extensions;
 
+      uint32_t pow_summary = 0;
```

Comment:
This is an ABI change for the storage, any late upgrades would break reading existing block_log files, and make the p2p protocol incompatible in an unsafe way.

Reply:
Oh, it doesn't yet, as it's not included in FC_REFLECT below, but it would need to be to make into transport & on storage.